src/client/lcd/api/IbcAPI.spec.ts

```diff
@@ -42,13 +42,11 @@ describe('IbcClientAPI', () => {
     expect(res).not.toBeUndefined();
   });
 
-  /*
   it('ica controller paramaters', async () => {
-    const res = await ibc.interchainAccountControllerParameters();
+    const res = await ibc.interchainAccountControllerParameters('pisco-1');
```

Comment:
i believe ica controller has been enabled in the recent 2.3 upgrade, but hitting `https://pisco-lcd.terra.dev/ibc/apps/interchain_accounts/controller/v1/params` gives me 
```{
  "code": 2,
  "message": "UnmarshalJSON cannot decode empty bytes: panic",
  "details": [
  ]
}
```

Reply:
but im able to pass the unit test, so let me know if we should make the change here